src/lucky_web/routeable.cr

```diff
@@ -23,28 +23,28 @@ module LuckyWeb::Routeable
     end
   {% end %}
 
-  macro nested_action
-    infer_nested_route
+  macro nested_action(singular = false)
+    infer_nested_route(singular: {{singular}})
 
     def call : LuckyWeb::Response
       {{yield}}
     end
   end
 
-  macro action
-    infer_route
+  macro action(singular = false)
+    infer_route(singular: {{singular}})
```

Comment:
Could you add some spaces inside the `{{  }}`? I haven't been consistent, but I'd like to do that from now on so it's easier to spot where interpolation is being done in macros

Reply:
Sure thing! I actually prefer the spaces, but most of the other places I checked didn't have them. I changed all the occurrences in `routeable.cr`, hope that's ok.